### ISSUE TITLE
Cannot read property 'display' of undefined

### ISSUE THREAD

**Issue by ex1st** [snapappointments/bootstrap-select#2086]
Version of bootstrap "3.3.7"
Version of bootstrap-select "1.13.2"

In BS3 `dropdown Constructor` doesn't have `Default` property.
That code throws error in some cases:
```javascript
display = that.options.display || $.fn.dropdown.Constructor.Default.display;
```

**Comment by ghost** [comment]
display = that.options.display || $.fn.dropdown.Constructor.Default.display;
==============>
display = that.options.display;

**Comment by jfcherng** [comment]
May be fixed by adding

```js
$.fn.selectpicker.Constructor.DEFAULTS.display = 'static'; // fix v1.13.2
```

But this should be fixed internally somehow.

**Comment by caseyjhol** [comment]
Released in [v1.13.3](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.3)!
